crates/aptos-crypto/src/ed25519/ed25519_keys.rs

```diff
@@ -38,6 +38,15 @@ impl Clone for Ed25519PrivateKey {
 #[derive(DeserializeKey, Clone, SerializeKey)]
 pub struct Ed25519PublicKey(pub(crate) ed25519_dalek::PublicKey);
 
+#[cfg(any(test, feature = "fuzzing"))]
```

Comment:
So this just picks arbitrary bytes, right?

Reply:
I think I should either move to a custom implementation of arbitrary or get rid of it. It comes in handy for starting a really dumb fuzzer without having to recreate too many types.